### ISSUE TITLE
Incorporating additional custom quality measure

### ISSUE THREAD

**Issue by yazhinia** [vtraag/leidenalg#186]
Hello,
      I would like to know if it is possible to combine existing modularity quality function with custom quality measure of community. If yes, could you provide an indication on how to do it and would it affect considerably speed of the algorithm?

Thank you.

**Comment by vtraag** [comment]
Yes, that is possible, this is described here: https://leidenalg.readthedocs.io/en/stable/implement.html
